config/constants.js

```diff
@@ -1,8 +1,19 @@
+'use strict';
+
 module.exports = {
-  AWS : {
-    BUCKET_NAME : 'my_bucket',
-    ACCESS_KEY_ID: 'ACCESS_KEY_ID',
-    SECRET_ACCESS_KEY: 'SECRET_ACCESS_KEY',
-    ENDPOINT: 'http://localhost:4567'
-  }
+  AWS: {
+    BUCKET_NAME: 'my_bucket',
+    ACCESS_KEY_ID: process.env.AWS_ACCESS_KEY || 'ACCESS_KEY_ID',
+    SECRET_ACCESS_KEY: process.env.AWS_SECRET_ACCESS_KEY || 'SECRET_ACCESS_KEY',
+    ENDPOINT: process.env.AWS_ENDPOINT || 'http://localhost:4567'
+  },
+  DB: {
+    URL: process.env.DB_URL || 'mongodb://localhost:27017/my_database'
```

Comment:
It will be more better if you can configure only the DB server address. you can create the db url with the address/localhost. Something like:- `url: 'mongodb://' + (process.env.DB_ADDRESS || 'localhost')`.